<!DOCTYPE html><html><head><title>Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for pharmacogenetics-guided warfarin dosing: 2017 update.</title><link rel="stylesheet" type="text/css" href="style.css"></head><body><div class='section'>
<h2>General
</h2>
<div class='field field_text'>
<h3>
Guideline Title
</h3>
<div class='FieldValue'>
<div class="content_title">Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for pharmacogenetics-guided warfarin dosing: 2017 update.</div>
</div>

</div>
<div class='field field_citation'>
<h3>
Bibliographic Source(s)
</h3>
<div class='FieldValue'>
<table><tr><td>Johnson JA, Caudle KE, Gong L, Whirl-Carrillo M, Stein CM, Scott SA, Lee MT, Gage BF, Kimmel SE, Perera MA, Anderson JL, Pirmohamed M, Klein TE, Limdi NA, Cavallari LH, Wadelius M. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for pharmacogenetics-guided warfarin dosing: 2017 update. Clin Pharmacol Ther. 2017 Sep;102(3):397-404. [46 references] <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=28198005 " target="_blank">PubMed</a></td></tr></table>
</div>

</div>
<div class='field field_text'>
<h3>
Guideline Status
</h3>
<div class='FieldValue'>
<div class="content_para"><p>This is the current release of the guideline. </p>
<p>This guideline meets NGC's 2013 (revised) inclusion criteria.</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>NEATS Assessment
</h2>
<div class='field field_picklist-one'>
<h3>
Disclosure of Guideline Funding Source
</h3>
<div class='FieldValue'>
Yes
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Disclosure and Management of Financial Conflict of Interests
</h3>
<div class='FieldValue'>
4
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Guideline Development Group Composition: Multidisciplinary Group
</h3>
<div class='FieldValue'>
Yes
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Guideline Development Group Composition: Methodologist Involvement
</h3>
<div class='FieldValue'>
Unknown
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Guideline Development Group Composition: Patient and Public Perspectives
</h3>
<div class='FieldValue'>
1
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Use of a Systematic Review of Evidence: Search Strategy
</h3>
<div class='FieldValue'>
4
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Use of a Systematic Review of Evidence: Study Selection
</h3>
<div class='FieldValue'>
4
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Use of a Systematic Review of Evidence: Synthesis of Evidence
</h3>
<div class='FieldValue'>
3
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Evidence Foundations for and Rating Strength of Recommendations: Grading the Quality or Strength of Evidence
</h3>
<div class='FieldValue'>
4
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Evidence Foundations for and Rating Strength of Recommendations: Benefits and Harms of Recommendations
</h3>
<div class='FieldValue'>
5
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Evidence Foundations for and Rating Strength of Recommendations: Evidence Summary Supporting Recommendations
</h3>
<div class='FieldValue'>
4
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Evidence Foundations for and Rating Strength of Recommendations: Rating the Strength of Recommendations
</h3>
<div class='FieldValue'>
5
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Specific and Unambiguous Articulation of Recommendations
</h3>
<div class='FieldValue'>
4
</div>

</div>
<div class='field field_picklist-one'>
<h3>
External Review
</h3>
<div class='FieldValue'>
2
</div>

</div>
<div class='field field_picklist-one'>
<h3>
Updating
</h3>
<div class='FieldValue'>
4
</div>

</div>

</div>
<div class='section'>
<h2>Recommendations
</h2>
<div class='field field_text'>
<h3>
Major Recommendations
</h3>
<div class='FieldValue'>
<div class="content_para"><p>The strength of therapeutic recommendations (<strong>Strong</strong>, <strong>Moderate</strong>, <strong>Optional</strong>) is defined at the end of the "Major Recommendations" field.</p>
<p><strong><span style="text-decoration: underline;">Genetic Test Interpretation</span></strong></p>
<p><strong>Cytochrome P450 2C9 (<em>CYP2C9</em>)</strong></p>
<p>Clinical laboratories typically report <em>CYP2C9</em> genotype results using the star (*) allele nomenclature system and an interpretation that includes a predicted metabolizer phenotype (<em>CYP2C9</em> allele definition table [see the "Availability of Companion Documents" field]). Most U.S. Food and Drug Administration (FDA)-approved <em>CYP2C9</em> tests include only <em>*2</em> and <em>*3</em>, which is not as informative for African ancestry populations; however, some clinical laboratories may offer expanded <em>CYP2C9</em> panels validated as laboratory developed tests (LDTs) (for allele frequencies, see <em>CYP2C9</em> frequency table [see the "Availability of Companion Documents" field]). </p>
<p><strong>Vitamin K Epoxide Reductase Complex Subunit 1 (<em>VKORC1</em>)</strong></p>
<p>Clinical laboratories typically report <em>VKORC1</em> genotype results by c.-1639G&gt;A (or the linked 1173C&gt;T; rs9934438) genotype (e.g., G/A) and an interpretation on warfarin sensitivity (<em>VKORC1</em> allele definition table [see the "Availability of Companion Documents" field]). Most commercial genotyping platforms do not detect rare <em>VKORC1</em> variants that have been associated with warfarin resistance (<em>VKORC1</em> frequency table [see the "Availability of Companion Documents" field]). </p>
<p><strong>Cytochrome P450 4F2 (<em>CYP4F2</em>)</strong></p>
<p>Although not as commonly tested for as <em>CYP2C9</em> and <em>VKORC1</em>, some clinical laboratories may also test for <em>CYP4F2</em> using a targeted genotyping laboratory developed test to detect <em>CYP4F2*3</em> (c.1297G&gt;A, p.Val433Met; rs2108622) variant (<em>CYP4F2</em> allele definition table [see the "Availability of Companion Documents" field]). Results are typically reported by nucleotide (e.g., G/A), amino acid (e.g., Val/Met) or star (*) allele (<em>*1/*3</em>) genotype and an interpretation related to warfarin dosing. </p>
<p><strong>CYP2C rs12777823</strong></p>
<p>Given the recent identification of the association between rs12777823 (g.96405502G&gt;A) and warfarin dosing among African Americans, most clinical laboratories do not currently include this non-coding variant in their warfarin pharmacogenetic genotyping panels. However, the increasing accessibility of clinical research genomics programs that return actionable results and the notable effect of this variant among African Americans suggests that some patients may have genotype results for this variant in the future. Results would likely be reported by genotype (e.g., G/A) and an interpretation related to warfarin dosing.</p>
<p><strong><span style="text-decoration: underline;">Genetic Test Options</span></strong></p>
<p>Commercially available genetic testing options change over time. Additional information about pharmacogenetic testing can be found at the Genetic Testing Registry (<a href="http://www.ncbi.nlm.nih.gov/gtr/" title="Genetic Testing Registry (GTRÂ®)" target="_blank">http://www.ncbi.nlm.nih.gov/gtr/</a>). </p>
<p><strong><span style="text-decoration: underline;">Incidental Findings</span></strong></p>
<p>No diseases have been linked to common <em>CYP2C9</em> variants independent of drug metabolism and response. Similarly, no diseases have been consistently linked to common <em>VKORC1</em> and <em>CYP4F2</em> variants that are interrogated in warfarin response tests. However, homozygosity for rare coding mutations in <em>VKORC1</em> are a known cause of combined deficiency of vitamin K-dependent clotting factors-2 (VKCFD2), which is a rare and potentially fatal bleeding disorder that can be reversed by oral administration of vitamin K.</p>
<p><strong><span style="text-decoration: underline;">Therapeutic Recommendations: Adults</span></strong></p>
<p><strong>Recommendations for Warfarin Maintenance (Chronic) Dosage Based on Genetic Information</strong></p>
<p>The authors use the three-tiered rating system, described in the "Rating Scheme for the Strength of the Recommendations" field, in which ratings of strong, moderate, and optional are applied based on the evidence reviewed. The recommendations for dosing based on genotype contained herein include recommendations and are derived from numerous observational and prospective studies, and randomized trials that suggest the ability to more accurately identify stable therapeutic warfarin dose requirements through the use of both genetic and clinical information. Data from prospective studies and randomized controlled trials are equivocal on whether the improvement in dosing prediction by pharmacogenetics dosing leads to improved clinical outcomes. The majority of the literature underpinning these guidelines arises from individuals of European ancestry, African Americans, and East Asians. However, the more limited literature in other populations generally suggests the guidelines are appropriate in them also. </p>
<p>Numerous studies have derived warfarin dosing algorithms that use both genetic and nongenetic factors to predict warfarin dose. Two algorithms perform well in estimating stable warfarin dose and were created using more than 5,000 subjects, although as noted above, more recent data suggest they do not perform acceptably in African Americans when used without modification for <em>CYP2C9</em> alleles frequently found in the African population. The Gage and International Warfarin Pharmacogenetics Consortium (IWPC) algorithms or minor adjustments to them have also been the algorithms used in both randomized controlled trials and most of the prospective dosing studies. Dosing algorithms using genetic information outperform nongenetic clinical algorithms and fixed-dose approaches in dose prediction, except in African Americans when the algorithm only includes <em>CYP2C9*2</em> and <em>*3</em>. Genetics-based algorithms also better predict warfarin dose than the FDA-approved warfarin label table.</p>
<p><strong>Pharmacogenetic Algorithm-based Warfarin Dosing</strong></p>
<p>This guideline recommends that pharmacogenetic warfarin dosing be accomplished through the use of one of the pharmacogenetic dosing algorithms described above, as summarized in Figure 2 in the original guideline document. These algorithms, as originally published, are available in the Supplement (see the "Availability of Companion Documents" field). The two algorithms provide very similar dose recommendations. The clinical and genetic information used in one or both algorithms is shown in Text Box 1 in the original guideline document. These algorithms compute the anticipated stable daily warfarin dose to one decimal and the clinician must then prescribe a regimen (e.g., an estimate of 4.3 mg/day might be given as 4 mg daily except 5 mg 2 days per week). An additional "dose revision" algorithm, which can be used on days 4&ndash;5 of therapy for dose refinement and uses genetic information, was tested in the COAG and EU-PACT trials and can also be used (Supplemental Table S5). </p>
<p>It is important to note that these algorithms do not include <em>CYP4F2</em>, <em>CYP2C9*5</em>, <em>*6</em>, <em>*8</em>, or <em>*11</em> or rs12777823, and incorporation of these should be added when results are available, as described in Figure 2. The <a href="http://www.warfarindosing.org/Source/Home.aspx" title="WarfarinDosing.org Web site">WarfarinDosing.org</a> contains both algorithms, the Gage algorithm as the primary algorithm and the IWPC algorithm as the secondary algorithm and can adjust for <em>CYP4F2</em>, <em>CYP2C9*5</em>, and <em>*6</em>. If utilizing <a href="http://www.warfarindosing.org/Source/Home.aspx" title="WarfarinDosing.org Web site">WarfarinDosing.org</a>, the user should be clear on whether the algorithm is or is not incorporating genotypes beyond <em>CYP2C9 *2</em> and <em>*3</em> and <em>VKORC1</em>, which are the only three genotypes in the original version of both algorithms.</p>
<p><strong>Pharmacogenetics-informed Loading (or Initiation) Dose Calculation</strong></p>
<p>The use of a different initial warfarin dose (or "loading dose") is somewhat controversial and plays different roles in different regions of the world, based on experience and local standards. Recent data from a diverse U.S.-based cohort suggest that failure to provide a loading dose in patients with zero or one variant alleles in <em>VKORC1</em> or <em>CYP2C9</em> may delay time to therapeutic INR and reduce time in therapeutic range in the initial month of therapy. A genetically guided loading dose approach was developed by Avery et al. and a slightly modified version was successfully implemented in the EU-PACT trial. In COAG <em>CYP2C9</em> variant alleles were not considered for the initial dose, providing a small loading dose on day 1. Whether differences in loading dose strategies between the EU-PACT and COAG trials contributed to differing results is not known. If loading doses are to be used, a genetically informed approach to calculating the loading dose may be helpful. The majority of the experience with a genetically informed loading regimen is in those of European ancestry. Determination of maintenance dose would be as described above. </p>
<p><strong>Non-African Ancestry Recommendation</strong></p>
<p>In patients who self-identify as non-African ancestry, the recommendation, as summarized in Figure 2, is to: 1) calculate warfarin dosing using a published pharmacogenetic algorithm, including genotype information for <em>VKORC1-1639G&gt;A</em> and <em>CYP2C9*2</em> and <em>*3</em>. In individuals with genotypes associated with CYP2C9 poor metabolism (e.g., <em>CYP2C9 *2/*3, *3/*3</em>) or both increased sensitivity (<em>VKORC1-1639</em> A/A) and CYP2C9 poor metabolism, an alternative oral anticoagulant might be considered. The bulk of the literature informing these recommendations is in European and Asian ancestry populations, but consistent data exist for other non-African populations. These recommendations are graded as <strong>STRONG</strong>. 2) If a loading dose is to be utilized, the EU-PACT loading dose algorithm that incorporates genetic information could be used. This recommendation is <strong>OPTIONAL</strong>. 3) While <em>CYP2C9*5</em>, <em>*6</em>, <em>*8</em>, or <em>*11</em> variant alleles are commonly referred to as African-specific alleles, they can occur among individuals who do not identify as, or know of their, African ancestry. If these variant alleles are detected, decrease calculated dose by 15% to 30% per variant allele or consider an alternative agent. Larger dose reductions might be needed in patients homozygous for variant alleles (i.e., 20% to 40%, e.g., <em>CYP2C9*2/*5</em>). This recommendation is graded as <strong>OPTIONAL</strong>. 4) If the <em>CYP4F2*3</em> (i.e., c.1297A, p.433Met) allele is also detected, increase the dose by 5% to 10%. This recommendation is also considered <strong>OPTIONAL</strong>. 5) The data do not suggest an association between rs12777823 genotype and warfarin dose in non-African Americans, thus rs12777823 should not be considered in these individuals (even if available). </p>
<p><strong>African Ancestry Recommendation</strong> </p>
<p>In patients of African ancestry, <em>CYP2C9*5</em>, <em>*6</em>, <em>*8</em>, <em>*11</em> are important for warfarin dosing. If these genotypes are not available, warfarin should be dosed clinically without consideration for genotype. If <em>CYP2C9*5</em>, <em>*6</em>, <em>*8</em>, and <em>*11</em> are known, then the recommendation, as shown in Figure 2, is to: 1) calculate warfarin dose using a validated pharmacogenetic algorithm, including genotype information for <em>VKORC1 c.-1639G&gt;A</em> and <em>CYP2C9*2</em> and <em>*3</em>; 2) if the individual carries a <em>CYP2C9*5</em>, <em>*6</em>, <em>*8</em>, or <em>*11</em> variant allele(s), decrease calculated dose by 15% to 30%. Larger dose reductions might be needed in patients who carry two variant alleles (e.g., <em>CYP2C9*5/*6</em>) (i.e., 20% to 40% dose reduction). 3) In addition, rs12777823 is associated with warfarin dosing in African Americans (mainly originating from West Africa). Thus, in African Americans a dose reductions of 10% to 25% in those with rs12777823 A/G or A/A genotype is recommended. These recommendations are considered <strong>MODERATE</strong>. </p>
<p>In individuals with genotypes that predict CYP2C9 poor metabolism or who have increased warfarin sensitivity (<em>VKORC1</em> c.-1639 A/A) and CYP2C9 poor metabolism, an alternative oral anticoagulant should be considered, for non-African ancestry. As noted above, if a loading dose is to be used, the EU-PACT algorithm that incorporates genetic information could be used to calculate loading dose. This recommendation is <strong>OPTIONAL</strong>. The data do not support an impact on clinical phenotype for <em>CYP4F2</em> on warfarin dosing in those of African ancestry and so no recommendation is made for use of <em>CYP4F2</em> genotype data in blacks.</p>
<p><strong><span style="text-decoration: underline;">Recommendations for Pediatric Patients</span></strong></p>
<p>As detailed in Supplemental Table S7, there is strong evidence for the use of <em>CYP2C9*2</em> and <em>*3</em> and <em>VKORC1-1639G&gt;A</em> genotype to guide warfarin dosing in children of European ancestry. The studies in Japanese pediatric individuals are conflicting, as <em>VKORC1</em> and <em>CYP2C9</em> could not be adequately evaluated due to the low numbers of <em>CYP2C9</em> variant carriers. For other ethnicities, there is no evidence documenting that <em>VKORC1</em> and <em>CYP2C9</em> are important. Furthermore, there are no data in children that included <em>CYP2C9*5</em>, <em>*6</em>, <em>*8</em>, or <em>*11</em> genotyping. Based on the current evidence, in children of European ancestry and if <em>CYP2C9*2</em> and <em>*3</em> and <em>VKORC1-1639</em> genotype are available, calculate warfarin dosing based on a validated published pediatric pharmacogenetic algorithm (see Figure 3 in the original guideline document). A dosing tool that can be used in children of European ancestry is available at <a href="http://www.warfarindoserevision.com" title="Warfarin Dose Revision Web site">http://www.warfarindoserevision.com</a>.</p>
<p><strong><span style="text-decoration: underline;">Other Considerations</span></strong> </p>
<p>Given the effects of CYP2C9 on warfarin clearance, and given that the <em>CYP2C9</em> variant alleles are associated with reduced warfarin clearance, <em>CYP2C9</em> genotype may influence time to onset and offset of anticoagulation, as measured by INR. The Supplemental Material summarizes other considerations in the dosing of warfarin, including clinical factors and interacting drugs, some of which are included in the pharmacogenetic dosing algorithms (see Text Box 1 in the original guideline document). Other genes of potential importance are detailed in the Supplemental Material and Supplemental Table S6, including <em>CALU</em> and <em>GGCX</em>. Most clinical genotyping platforms do not include these genes, nor do the dosing tables or published algorithms. The Supplemental Material also discusses incorporation of genetic information into the initial dose, and alternatives to warfarin.</p>
<p><strong><span style="text-decoration: underline;">Definitions</span></strong></p>
<p><strong>Strength of Therapeutic Recommendations</strong></p>
<p><strong>Strong</strong>: The evidence is high quality and the desirable effects clearly outweigh the undesirable effects.</p>
<p><strong>Moderate</strong>: There is a close or uncertain balance as to whether the evidence is high quality and the desirable clearly outweigh the undesirable effects.</p>
<p><strong>Optional</strong>: The desirable effects are closely balanced with undesirable effects and there is room for differences of opinion as to the need for the recommended course of action.</p>
<p><strong>No recommendation</strong>: There is insufficient evidence, confidence, or agreement to provide a recommendation to guide clinical practice at this time.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Clinical Algorithm(s)
</h3>
<div class='FieldValue'>
<div class="content_para"><p>The following algorithms are provided in the original guideline document:</p>
<ul style="list-style-type: disc;">
    <li>Dosing recommendations for warfarin dosing based on genotype for adult patients </li>
    <li>Dosing recommendations for warfarin dosing based on genotype for pediatric patients </li>
</ul>
<p>Specific dosing algorithms are provided in the supplemental material (see the "Availability of Companion Documents" field).</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Scope
</h2>
<div class='field field_text'>
<h3>
Disease/Condition(s)
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Thromboembolic disorders</p></div>
</div>

</div>
<div class='field field_picklist-many'>
<h3>
Guideline Category
</h3>
<div class='FieldValue'>
Management
</div>
<div class='FieldValue'>
Prevention
</div>
<div class='FieldValue'>
Treatment
</div>

</div>
<div class='field field_picklist-many'>
<h3>
Clinical Specialty
</h3>
<div class='FieldValue'>
Cardiology
</div>
<div class='FieldValue'>
Internal Medicine
</div>
<div class='FieldValue'>
Medical Genetics
</div>
<div class='FieldValue'>
Pediatrics
</div>
<div class='FieldValue'>
Pharmacology
</div>
<div class='FieldValue'>
Pulmonary Medicine
</div>

</div>
<div class='field field_picklist-many'>
<h3>
Intended Users
</h3>
<div class='FieldValue'>
Advanced Practice Nurses
</div>
<div class='FieldValue'>
Nurses
</div>
<div class='FieldValue'>
Pharmacists
</div>
<div class='FieldValue'>
Physician Assistants
</div>
<div class='FieldValue'>
Physicians
</div>

</div>
<div class='field field_text'>
<h3>
Guideline Objective(s)
</h3>
<div class='FieldValue'>
<div class="content_para"><p>To assist in the interpretation and use of cytochrome P450 2C9 (<em>CYP2C9</em>), vitamin K epoxide reductase complex subunit 1 (<em>VKORC1</em>), cytochrome P450 4F2 (<em>CYP4F2</em>), and rs12777823 genotypes to estimate therapeutic warfarin dose among patients with a target international normalized ratio (INR) of 2&ndash;3, should clinical genotype results be available to the clinician</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Target Population
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Patients requiring management or treatment of thromboembolic disorders</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Interventions and Practices Considered
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Use of cytochrome P450 2C9 (<em>CYP2C9</em>), vitamin K epoxide reductase complex subunit 1 (<em>VKORC1</em>), cytochrome P450 4F2 (<em>CYP4F2</em>), and the <em>CYP2C</em> cluster (e.g., rs12777823) genotyping to guide use of warfarin</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Major Outcomes Considered
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Effect of cytochrome P450 2C9 (<em>CYP2C9</em>), vitamin K epoxide reductase complex subunit 1 (<em>VKORC1</em>), cytochrome P450 4F2 (<em>CYP4F2</em>), and the <em>CYP2C</em> cluster (e.g., rs12777823) on warfarin clinical outcomes or effect on pharmacokinetics</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Methodology
</h2>
<div class='field field_picklist-many'>
<h3>
Methods Used to Collect/Select the Evidence
</h3>
<div class='FieldValue'>
Hand-searches of Published Literature (Primary Sources)
</div>
<div class='FieldValue'>
Hand-searches of Published Literature (Secondary Sources)
</div>
<div class='FieldValue'>
Searches of Electronic Databases
</div>

</div>
<div class='field field_text'>
<h3>
Description of Methods Used to Collect/Select the Evidence
</h3>
<div class='FieldValue'>
<div class="content_para"><p><strong><span style="text-decoration: underline;">Literature Review</span></strong></p>
<p>The authors searched the PubMed&reg; database (January 1966 to August 2016) for the following keywords: ((cytochrome P450 2C9 or CYP2C9) OR (VKORC1) OR (cytochrome P450 4F2 or CYP4F2) AND (warfarin) AND English [Language]). Using these search terms, 1221 publications were identified. In addition, studies annotated in PharmGKB (<a href="http://www.pharmgkb.org" title="PharmGKB Web site">http://www.pharmgkb.org</a>) were identified. Study inclusion criteria included publications that included analyses for the association between <em>CYP2C9/VKORC1/CYP4F</em>2 genotypes and metabolism of warfarin or warfarin-related adverse drug events or clinical outcomes. Non-English manuscripts were excluded.</p>
<p>The <em>CYP2C9/VKORC1/CYP4F2</em> frequency tables (see the "Availability of Companion Documents" field) were made by searching the PubMed&reg; database (1995 to 2016). The following criteria were used for <em>CYP2C9</em>: (CYP2C9 or 2C9 or cytochrome P4502C9) AND (genotype OR allele OR frequency OR minor allele OR variant OR ethnic OR race OR racial OR ethnicity OR population) with filter limits set to retrieve "full-text" and "English" literature. In addition, reports were also identified from citations by others or review articles. Studies were considered for inclusion in the <em>CYP2C9</em> frequency table if: (1) the ethnicity of the population was clearly indicated, (2) either allele frequencies or genotype frequencies were reported, (3) the method by which the genes were genotyped was indicated, (4) the sample population consisted of at least 50 individuals with a few exceptions (e.g., smaller cohorts that were part of larger studies) and (5) the study represented an original publication (no reviews or meta-analyses). Similar search strategies were used for <em>VKORC1</em> and <em>CYP4F2</em> genes. Allele frequencies reported in phase 3 1000 Genomes were also included (<a href="http://phase3browser.1000genomes.org/index.html" title="1000 Genomes Web site">http://phase3browser.1000genomes.org/index.html</a>).</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Number of Source Documents
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Following application of the inclusion criteria, 151 publications were reviewed and included in the evidence tables (Supplemental Tables S1- S7 [see the "Availability of Companion Documents" field]).</p></div>
</div>

</div>
<div class='field field_picklist-choice'>
<h3>
Methods Used to Assess the Quality and Strength of the Evidence
</h3>
<div class='FieldValue'>
Weighting According to a Rating Scheme (Scheme Given)
</div>

</div>
<div class='field field_text'>
<h3>
Rating Scheme for the Strength of the Evidence
</h3>
<div class='FieldValue'>
<div class="content_para"><p><strong><span style="text-decoration: underline;">Levels of Evidence Linking Genotype to Phenotype</span></strong></p>
<p><strong>High</strong>: Evidence includes consistent results from well-designed, well-conducted studies.</p>
<p><strong>Moderate</strong>: Evidence is sufficient to determine effects, but the strength of the evidence is limited by the number, quality, or consistency of the individual studies; generalizability to routine practice; or indirect nature of the evidence.</p>
<p><strong>Weak</strong>: Evidence is insufficient to assess the effects on health outcomes because of limited number or power of studies, important flaws in their design or conduct, gaps in the chain of evidence, or lack of information.</p></div>
</div>

</div>
<div class='field field_picklist-many'>
<h3>
Methods Used to Analyze the Evidence
</h3>
<div class='FieldValue'>
Review of Published Meta-Analyses
</div>
<div class='FieldValue'>
Systematic Review with Evidence Tables
</div>

</div>
<div class='field field_text'>
<h3>
Description of the Methods Used to Analyze the Evidence
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Some of the factors that are taken into account in evaluating the evidence supporting therapeutic recommendations include: <em>in vivo</em> pharmacokinetic and pharmacodynamic data, in vitro enzyme activity of tissues expressing wild-type or variant-containing gene, <em>in vitro</em> enzyme activity from tissues isolated from individuals of known genotypes, and <em>in vivo</em> pre-clinical and clinical pharmacokinetic and pharmacodynamic studies. The gene-based dosing recommendations in this guideline take into consideration the effects cytochrome P450 2C9 (<em>CYP2C9</em>)/vitamin K epoxide reductase complex subunit 1 (<em>VKORC1</em>)/cytochrome P450 4F2 (<em>CYP4F2</em>) genetic variants may have on both clinical outcomes and warfarin pharmacokinetics.</p>
<p><strong><span style="text-decoration: underline;">Summarization and Presentation of the Evidence Linking Genotype to Drug Variability</span></strong></p>
<p>Publications supporting a major finding are usually considered as a group and scored by members of the writing committee based on the totality of the evidence supporting that major finding. Thus, it is possible for an evidentiary conclusion based on many papers, each of which may be relatively weak, to be graded as "moderate" or even "strong," if there are multiple small case reports or studies that are all supportive with no contradictory studies. The rating scheme (see the "Rating Scheme for the Strength of the Evidence" field) uses a scale modified slightly from Valdes et al. Primary publications are summarized in the Evidence Table which is published in the manuscript supplemental material (see the "Availability of Companion Documents" field). It is the writing committee's evaluation of this evidence that provides the basis for the therapeutic recommendation(s). </p></div>
</div>

</div>
<div class='field field_picklist-choice'>
<h3>
Methods Used to Formulate the Recommendations
</h3>
<div class='FieldValue'>
Expert Consensus
</div>

</div>
<div class='field field_text'>
<h3>
Description of Methods Used to Formulate the Recommendations
</h3>
<div class='FieldValue'>
<div class="content_para"><p><strong><span style="text-decoration: underline;">Identification of Content Experts and Formation of Writing Committee</span></strong></p>
<p>Once a guideline topic has been approved by Clinical Pharmacogenetics Implementation Consortium (CPIC) members and the Steering Committee, a senior author is identified through self-nomination or by request of the CPIC Steering Committee. The senior author takes responsibility for forming the writing committee and completing the guideline. The writing committee is multidisciplinary, comprising a variety of scientists, pharmacologists, and clinicians (e.g., pharmacists and physicians). Authors will have a track record of publication and/or expertise in the specific topic area of the guideline. PharmGKB assigns at least one Scientific Curator to each CPIC guideline writing committee who has expertise in searching, compiling and evaluating the evidence for gene-drug associations, and making it computable and available on the PharmGKB Web site. Furthermore, PharmGKB curators often take primary responsibility for completing background gene and drug summaries, assigning likely phenotypes based on genotypes (i.e., "Table 1" in guidelines), literature review, as well as preparing supplementary material provided in each guideline (i.e., genotypes that constitute the star (*) alleles or haplotypes, frequencies of alleles in major race/ethnic groups, genetic test interpretation and availability, and evidence linking genotype with phenotype).</p>
<p><strong><span style="text-decoration: underline;">Development of Therapeutic Recommendation and Assignment of Strength of the Recommendation</span></strong></p>
<p>The writing committee discusses the evaluation of the literature and develops a draft recommendation via Web conferences and email communication. CPIC's therapeutic recommendations are based on weighing the evidence summarized in the supplementary Evidence Table from a combination of preclinical functional and clinical data, as well as on any existing consensus guidelines. Evidence related to the suitability of alternative medications or dosing that may be used based on genetics must be weighed in assigning the strength of the recommendation. Overall, the therapeutic recommendations are simplified to allow rapid interpretation by clinicians and are presented in the Table 2 of each guideline and occasionally in an algorithm. </p>
<p>To assign strength to a recommendation, CPIC uses a transparent three category system (see the "Rating Scheme for the Strength of the Recommendations" field) for rating recommendations that was adopted with slight modification from the rating scale for evidence-based recommendations on the use of antiretroviral agents (<a href="http://aidsinfo.nih.gov/contentfiles/AdultandAdolescentGL.pdf" title="AIDSinfo Web site" target="_blank">http://aidsinfo.nih.gov/contentfiles/AdultandAdolescentGL.pdf</a>). Each recommendation also includes an assessment of its usefulness in pediatric patients. </p>
<p>CPIC guidelines currently focus on gene-drug pairs for which at least one of the prescribing recommendations is actionable (e.g., recommendation to alter a dose or consider an alternative drug based on the genotype-phenotype relationship). For these and many other gene-drug pairs, PharmGKB also contains clinical annotations that are genotype-based summaries of the association between a drug and a particular variant. Each clinical annotation is assigned a level of evidence depending on population, replication, effect size and statistical significance.</p>
<p>Refer to "Incorporation of pharmacogenomics into routine clinical practice: the Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline development process" (see the "Availability of Companion Documents" field) for additional information. </p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Rating Scheme for the Strength of the Recommendations
</h3>
<div class='FieldValue'>
<div class="content_para"><p><strong><span style="text-decoration: underline;">Strength of Therapeutic Recommendations</span></strong></p>
<p><strong>Strong</strong>: The evidence is high quality and the desirable effects clearly outweigh the undesirable effects.</p>
<p><strong>Moderate</strong>: There is a close or uncertain balance as to whether the evidence is high quality and the desirable clearly outweigh the undesirable effects.</p>
<p><strong>Optional</strong>: The desirable effects are closely balanced with undesirable effects and there is room for differences of opinion as to the need for the recommended course of action.</p>
<p><strong>No recommendation</strong>: There is insufficient evidence, confidence, or agreement to provide a recommendation to guide clinical practice at this time.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Cost Analysis
</h3>
<div class='FieldValue'>
<div class="content_para"><p>The cost-benefit of genetic-guided therapy depends on the cost of genotyping and the reduction in adverse events, and most insurance plans do not currently pay for warfarin pharmacogenetic testing.</p></div>
</div>

</div>
<div class='field field_picklist-choice'>
<h3>
Method of Guideline Validation
</h3>
<div class='FieldValue'>
External Peer Review
</div>
<div class='FieldValue'>
Internal Peer Review
</div>

</div>
<div class='field field_text'>
<h3>
Description of Method of Guideline Validation
</h3>
<div class='FieldValue'>
<div class="content_para"><p><span style="text-decoration: underline;"><strong>Internal and External Review, Comment, and Approval Process</strong></span> </p>
<p>Once the writing committee has completed and approved a draft guideline, the draft guideline is circulated to the Clinical Pharmacogenetics Implementation Consortium (CPIC) co-leaders and coordinator for content review. The guideline is reviewed for compliance with the CPIC Standard Operating Procedures and required format. The guideline draft is then discussed on a CPIC conference call with all CPIC members and circulated to the members for further review and approval. At each stage, feedback is considered for incorporation into the guideline and/or revision of the guideline, as supported by the available evidence and expert clinical judgment of the senior author and writing committee. Finally, the guideline manuscript undergoes typical external scientific peer review by the journal prior to publication. Current agreements with the American Society for <em>Clinical Pharmacology and Therapeutics</em> give the journal Clinical Pharmacology and Therapeutics the first right of refusal for publication of CPIC guidelines; as part of this agreement, the guidelines are freely posted to PharmGKB immediately upon publication. In general <em>Clinical Pharmacology and Therapeutics</em> uses a minimum of two external expert peer-reviewers and an editorial board member with content expertise as reviewers for each CPIC guideline.</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Evidence Supporting the Recommendations
</h2>
<div class='field field_text'>
<h3>
Type of Evidence Supporting the Recommendations
</h3>
<div class='FieldValue'>
<div class="content_para"><p>The evidence summarized in Supplemental Tables S1 to S7 (see the "Availability of Companion Documents" field) is graded on a scale of high, moderate, and weak, based upon the level of evidence (see the Rating Scheme for the Strength of the Evidence" field). Every effort was made to present evidence from high-quality studies.</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Benefits/Harms of Implementing the Guideline Recommendations
</h2>
<div class='field field_text'>
<h3>
Potential Benefits
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Incorporation of genetic information has the potential to shorten the time to stable international normalized ratio (INR), increase the time within the therapeutic INR range, and reduce underdosing or overdosing during the initial treatment period. If these benefits are achieved, they could result in a reduced risk of bleeding and thromboembolic events.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Potential Harms
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Using genetic information to guide dosing may lead to false security and inadequate international normalized ratio (INR) monitoring. In particular, there are risks of using pharmacogenetic dosing in those of African ancestry if only cytochrome P450 2C9 (<em>CYP2C9</em>) <em>*2</em> and <em>*3</em> alleles are included. Genetic-guided dosing may increase the risk for overdosing or underdosing, especially in individuals who carry rare or untested variants and are assigned as "wild-type" by default. Although there is substantial evidence associating <em>CYP2C9</em> and vitamin K epoxide reductase complex subunit 1 (<em>VKORC1</em>) variants with warfarin dosing, randomized clinical trials have demonstrated inconsistent results in terms of clinical outcomes (see "Linking genetic variability to variability in drug-related phenotypes" in the original guideline document). Although genotyping is reliable when performed in qualified laboratories, an additional risk is an error in genotyping or reporting of genotype. Genotypes are life-long test results, so such error could have long-term adverse health implications.</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Qualifying Statements
</h2>
<div class='field field_text'>
<h3>
Qualifying Statements
</h3>
<div class='FieldValue'>
<div class="content_para"><p><span style="text-decoration: underline;"><strong>Caveats: Appropriate Use and/or Potential Misuse of Genetic Tests</strong></span></p>
<p>Many pharmacogenetic dosing algorithms are developed for a target international normalized ratio (INR) of 2&ndash;3 and so their utility for estimating therapeutic warfarin doses with other target INR ranges is uncertain; however, some algorithms accommodate the target INR explicitly. Pharmacogenetic-guided warfarin dosing does not alter the requirements for regular INR monitoring. There are patients for whom genetic testing is likely to be of little or no benefit, including those who already have had long-term treatment with stable warfarin doses and those who are unable to achieve stable dosing due to variable adherence. The greatest potential benefit is early in the course of therapy (before therapy initiation or in the early days of therapy). It is likely that patients on therapy for many weeks to months, with careful INR monitoring, will derive little benefit from subsequent warfarin pharmacogenetics testing.</p>
<p><strong><span style="text-decoration: underline;">Disclaimer</span></strong> </p>
<p>Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines reflect expert consensus based on clinical evidence and peer-reviewed literature available at the time they are written and are intended only to assist clinicians in decision-making and to identify questions for further research. New evidence may have emerged since the time a guideline was submitted for publication. Guidelines are limited in scope and are not applicable to interventions or diseases not specifically identified. Guidelines do not account for all individual variations among patients and cannot be considered inclusive of all proper methods of care or exclusive of other treatments. It remains the responsibility of the healthcare provider to determine the best course of treatment for a patient. Adherence to any guideline is voluntary, with the ultimate determination regarding its application to be made solely by the clinician and the patient. CPIC assumes no responsibility for any injury to persons or damage to persons or property arising out of or related to any use of CPIC's guidelines, or for any errors or omissions.</p>
<p><strong><span style="text-decoration: underline;">Underlying Assumption</span></strong> </p>
<p>The key underlying assumption for all CPIC guidelines is that clinical high-throughput and pre-emptive genotyping will eventually become common practice and clinicians will increasingly have patients' genotypes available before a prescription is written. Therefore, CPIC guidelines are designed to provide guidance to clinicians as to how available genetic test results should be interpreted to ultimately improve drug therapy, rather than to provide guidance as to whether a genetic test should or should not be ordered.</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Implementation of the Guideline
</h2>
<div class='field field_text'>
<h3>
Description of Implementation Strategy
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Refer to "Incorporation of pharmacogenomics into routine clinical practice: the Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline development process" (see the "Availability of Companion Documents" field) for information on guideline dissemination and connecting the guidelines to practice.</p></div>
</div>

</div>
<div class='field field_picklist-many'>
<h3>
Implementation Tools
</h3>
<div class='FieldValue'>
Clinical Algorithm
</div>
<div class='FieldValue'>
Resources
</div>

</div>

</div>
<div class='section'>
<h2>Institute of Medicine (IOM) National Healthcare Quality Report Categories
</h2>
<div class='field field_picklist-many'>
<h3>
IOM Care Need
</h3>
<div class='FieldValue'>
Getting Better
</div>
<div class='FieldValue'>
Staying Healthy
</div>

</div>
<div class='field field_picklist-many'>
<h3>
IOM Domain
</h3>
<div class='FieldValue'>
Effectiveness
</div>
<div class='FieldValue'>
Patient-centeredness
</div>

</div>

</div>
<div class='section'>
<h2>Identifying Information and Availability
</h2>
<div class='field field_citation'>
<h3>
Bibliographic Source(s)
</h3>
<div class='FieldValue'>
<table><tr><td>Johnson JA, Caudle KE, Gong L, Whirl-Carrillo M, Stein CM, Scott SA, Lee MT, Gage BF, Kimmel SE, Perera MA, Anderson JL, Pirmohamed M, Klein TE, Limdi NA, Cavallari LH, Wadelius M. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for pharmacogenetics-guided warfarin dosing: 2017 update. Clin Pharmacol Ther. 2017 Sep;102(3):397-404. [46 references] <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=28198005 " target="_blank">PubMed</a></td></tr></table>
</div>

</div>
<div class='field field_text'>
<h3>
Adaptation
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Not applicable: The guideline was not adapted from another source.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Date Released
</h3>
<div class='FieldValue'>
2017 Sep
</div>

</div>
<div class='field field_orglist-p'>
<h3>
Guideline Developer(s)
</h3>
<div class='FieldValue'>
Clinical Pharmacogenetics Implementation Consortium - Independent Expert Panel
</div>

</div>
<div class='field field_text'>
<h3>
Source(s) of Funding
</h3>
<div class='FieldValue'>
<div class="content_para"><p>This work was funded by the National Institutes of Health (NIH) for the Clinical Pharmacogenetics Implementation Consortium (CPIC) (R24GM115264) and PharmGKB (R24GM61374). Relevant funding for other authors includes: NIH grants U01 GM074492 and U01 HG 007269 (JAJ), K23GM104401 (SAS), GM109145 (CMS), R01HL092173 and K24HL133373 (NAL), R01 HL097036 (BFG) and grants from the Swedish Research Council (Medicine 521-2011-2440 and 521-2014-3370), the Swedish Heart and Lung Foundation (20120557 and 20140291), and the Thur&eacute;us' Foundation (MW), the UK Department of Health and Medical Research Council (MP), and the EU FP7 programme for funding EU-PACT (MP, MW).</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Guideline Committee
</h3>
<div class='FieldValue'>
<div class="content_para"><p>The Writing Committee</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Composition of Group That Authored the Guideline
</h3>
<div class='FieldValue'>
<div class="content_para"><p><em>Authors</em>: JA Johnson, Department of Pharmacotherapy and Translational Research, College of Pharmacy, and Center for Pharmacogenomics, University of Florida, Gainesville, Florida, USA; KE Caudle, Department of Pharmaceutical Sciences, St. Jude Children's Research Hospital, Memphis, Tennessee, USA; L Gong, Department of Biomedical Data Science, Stanford University, Stanford, California, USA; M Whirl-Carrillo, Department of Biomedical Data Science, Stanford University, Stanford, California, USA; CM Stein, Division of Clinical Pharmacology, Vanderbilt Medical School, Nashville, Tennessee, USA; SA Scott, Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, New York, USA; MT Lee, Laboratory for International Alliance on Genomic Research, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan, and National Center for Genome Medicine, and Institute of Biomedical Sciences, Academia Sinica, Taipei, Taiwan, and Genomic Medicine Institute, Geisinger Health System, Danville, Pennsylvania, USA; BF Gage, Department of Internal Medicine, Washington University in St. Louis, St. Louis, Missouri, USA; SE Kimmel, Center for Clinical Epidemiology and Biostatistics, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, USA, and Department of Medicine and Department of Biostatistics and Epidemiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, USA; MA Perera, Department of Medicine, University of Chicago, Chicago, Illinois, USA; JL Anderson, Intermountain Heart Institute, Intermountain Medical Center, and Department of Internal Medicine (Cardiology), University of Utah School of Medicine, Salt Lake City, Utah, USA; M Pirmohamed, Department of Molecular and Clinical Pharmacology, The Wolfson Centre for Personalised Medicine, Institute of Translational Medicine, University of Liverpool, Liverpool, UK; TE Klein, Department of Biomedical Data Science, Stanford University, Stanford, California, USA; NA Limdi, Department of Neurology and Epidemiology, University of Alabama at Birmingham, Birmingham, Alabama, USA; LH Cavallari, Department of Pharmacotherapy and Translational Research, College of Pharmacy, and Center for Pharmacogenomics, University of Florida, Gainesville, Florida, USA; M Wadelius, Department of Medical Sciences, Clinical Pharmacology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Financial Disclosures/Conflicts of Interest
</h3>
<div class='FieldValue'>
<div class="content_para"><p><strong><span style="text-decoration: underline;">Conflicts of Interest</span></strong></p>
<p>J.A.J. is on the Clinical Pharmacogenetics Implementation Consortium (CPIC) Steering Committee and has no conflicts of interest related to this guideline. T.E.K and M.W.C. are paid scientific advisors to the Rxight Pharmacogenetic Program. S.A.S. is a director of a clinical laboratory that performs <em>CYP2C9</em> and <em>VKORC1</em> genetic testing. All other authors declare no conflicts of interest related to this guideline.</p>
<p><strong><span style="text-decoration: underline;">Management of Conflicts of Interest</span></strong></p>
<p>All authors must declare any funding interests and activities potentially resulting in conflict of interest by written disclosure to the CPIC Steering Committee and writing committee before the approval of the authorship plan. Included are all possible conflicts including spouses/family members in declarations, National Institutes of Health (NIH) funding that could be interpreted to indicate that authors are "advocates" of the recommendations, as well as any sources of revenue from consulting, patents, stock ownership, etc. All conflicts of interest are reported in the guideline manuscript.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Guideline Status
</h3>
<div class='FieldValue'>
<div class="content_para"><p>This is the current release of the guideline. </p>
<p>This guideline meets NGC's 2013 (revised) inclusion criteria.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Guideline Availability
</h3>
<div class='FieldValue'>
<div class="content_para"><p>Available from the <a href="https://cpicpgx.org/content/guideline/publication/warfarin/2017/warfarin.pdf" title="Clinical Pharmacogenetics Implementation Consortium (CPIC) Web site">Clinical Pharmacogenetics Implementation Consortium (CPIC) Web site</a>.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Availability of Companion Documents
</h3>
<div class='FieldValue'>
<div class="content_para"><p>The following are available:</p>
<ul style="list-style-type: disc;">
    <li>Supplementary material, including tables and methodological information, is available from the <a href="https://cpicpgx.org/guidelines/guideline-for-warfarin-and-cyp2c9-and-vkorc1/" title="Clinical Pharmacogenetics Implementation Consortium (CPIC) Web site">Clinical Pharmacogenetics Implementation Consortium (CPIC) Web site</a>. </li>
    <li>A variety of resources, including allele definition, frequency, and functionality tables, drug resource mappings, and gene resource mapping are available from the <a href="https://cpicpgx.org/guidelines/guideline-for-warfarin-and-cyp2c9-and-vkorc1/" title="CPIC Web site">CPIC Web site</a>. </li>
    <li>Caudle KE, Klein TE, Hoffman JM, et al. Incorporation of pharmacogenomics into routine clinical practice: the Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline development process. Curr Drug Metab. 2014;15(2):209-17. Available from the <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3977533/pdf/CDM-15-209.pdf/" title="National Center for Biotechnology Information (NCBI) Web site" target="_blank">National Center for Biotechnology Information (NCBI) Web site</a>. </li>
</ul></div>
</div>

</div>
<div class='field field_text'>
<h3>
Patient Resources
</h3>
<div class='FieldValue'>
<div class="content_para"><p>None available</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
NGC Status
</h3>
<div class='FieldValue'>
<div class="content_para"><p>This NGC summary was completed by ECRI Institute on November 1, 2017. The information was verified by the guideline developer on December 5, 2017. </p>
<p>This NEATS assessment was completed by ECRI Institute on October 30, 2017. The information was verified by the guideline developer on December 5, 2017.</p></div>
</div>

</div>
<div class='field field_text'>
<h3>
Copyright Statement
</h3>
<div class='FieldValue'>
<div class="content_para"><p>This NGC summary is based on the original guideline, which is subject to the guideline developer's copyright restrictions.</p>
<p>CPIC is a shared project between PharmGKB and the Pharmacogenomics Research Network. This guideline is published in Clinical Pharmacology and Therapeutics (CPT), an American Society for Clinical Pharmacology and Therapeutics journal (ASCPT). Electronic access to the CPIC guidelines is available at PharmGKB, <a href="http://cpicpgx.org" title="CPIC Web site">cpicpgx.org</a>, and through PubMed Central.</p>
<p>CPIC&reg; is a registered service mark of the <a href="http://www.hhs.gov/" title="DHHS Web site" target="_blank">U.S. Department of Health &amp; Human Services (HHS)</a>.</p></div>
</div>

</div>

</div>
<div class='section'>
<h2>Disclaimer
</h2>
<div class='field field_text'>
<h3>
NGC Disclaimer
</h3>
<div class='FieldValue'>
<p>The National Guideline Clearinghouseâ¢ (NGC) does not develop, produce, approve, or endorse the guidelines represented on this site.</p> 
<p>All guidelines summarized by NGC and hosted on our site are produced under the auspices of medical specialty societies, relevant professional associations, public or private organizations, other government agencies, health care organizations or plans, and similar entities.</p> 
<p>Guidelines represented on the NGC Web site are submitted by guideline developers, and are screened solely to determine that they meet the <a href="/help-and-about/summaries/inclusion-criteria">NGC Inclusion Criteria</a>.</p> 
<p>NGC, AHRQ, and its contractor ECRI Institute make no warranties concerning the content or clinical efficacy or effectiveness of the clinical practice guidelines and related materials represented on this site. Moreover, the views and opinions of developers or authors of guidelines represented on this site do not necessarily state or reflect those of NGC, AHRQ, or its contractor ECRI Institute, and inclusion or hosting of guidelines in NGC may not be used for advertising or commercial endorsement purposes.</p> 
<p>Readers with questions regarding guideline content are directed to contact the guideline developer.</p>
</div>

</div>

</div>
</body></html>